File: themes/OrchidFutureImperfect/src/main/java/com/eden/orchid/html5up/futureimperfect/FutureImperfectModule.java
package com.eden.orchid.html5up.futureimperfect;

import com.eden.orchid.api.registration.OrchidModule;
import com.eden.orchid.api.theme.Theme;

public class FutureImperfectModule extends OrchidModule {

    @Override
    protected void configure() {
        addToSet(Theme.class, FutureImperfectTheme.class);
    }
}

File: themes/OrchidFutureImperfect/src/main/kotlin/com/eden/orchid/html5up/futureimperfect/FutureImperfectModule.kt
package com.eden.orchid.html5up.futureimperfect

import com.eden.orchid.api.registration.OrchidModule
import com.eden.orchid.api.theme.Theme

class FutureImperfectModule : OrchidModule() {

    override fun configure() {
        addToSet(Theme::class.java, FutureImperfectTheme::class.java)
    }
}
